services/QuillLMS/db/migrate/20210920131932_create_activity_survey_responses.rb

```diff
@@ -0,0 +1,12 @@
+class CreateActivitySurveyResponses < ActiveRecord::Migration[5.1]
+  def change
+    create_table :activity_survey_responses do |t|
+      t.references :activity_session, index: true, foreign_key: true
+      t.integer :emoji_selection, null: false
+      t.string :multiple_choice_selections, array: true, null: false
```

Comment:
For arrays, I usually use `default: []`.  Not sure if this is equivalent to `null: false` in this case, but thought it would be worth mentioning.